### ISSUE TITLE
site icon not displayed at tab

### ISSUE THREAD

**Issue by lyghter** [tiborbarsi/image-video-block-browser-addon#18]
Firefox 64.0.2
Image Video Block 3.4

If the extension was enabled before the page was loaded, the site icon is not displayed on the tab. [Screenshot](https://yadi.sk/d/nnaDVzRI8kDdpQ)

**Comment by rwp0** [comment]
Yes indeed, using the latest Firefox developer edition (140.0a1 (2025-05-09) (64-bit)).

![Image](https://github.com/user-attachments/assets/76a87a7a-e282-41d8-847d-83a866abdf9f)

@tiborbarsi could you please review this (old) issue

Thank you very much for this extension, it's very useful and efficient in blocking various kinds of media, except that I don't want to block website icons, else I won't be able to identify my tabs among a long list thereof.

By "Block Images" option we usually don't intend to block favicons as well, I think.

Thanks
